index.js

```diff
@@ -1,15 +1,15 @@
 // Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
 
-import Log from './src/Log';
-import OidcClient from './src/OidcClient';
-import WebStorageStateStore from './src/WebStorageStateStore';
-import InMemoryWebStorage from './src/InMemoryWebStorage';
-import UserManager from './src/UserManager';
-import AccessTokenEvents from './src/AccessTokenEvents';
-import MetadataService from './src/MetadataService';
+export const Log = require('./src/Log');
+export const OidcClient = require('./src/OidcClient');
+export const WebStorageStateStore = require('./src/WebStorageStateStore');
+export const InMemoryWebStorage = require('./src/InMemoryWebStorage');
+export const UserManager = require('./src/UserManager');
+export const AccessTokenEvents = require('./src/AccessTokenEvents');
+export const MetadataService = require('./src/MetadataService');
 
-export {
+export default {
     Log,
     OidcClient,
     WebStorageStateStore,
```

Comment:
No - this is the entry file for both webpack configs. From there webpack will crawl the `require` and `import` statements to gather the required modules and bundle them together into the file written to `dist` (`npm run build` & `npm run build:classic`). I've made changes to the `import` statements because in ES6 you cannot write something like this:

    export const UserManager = import UserManager from './src/UserManager';

That's pretty much the only reason. 

Anyone downloading this via npm and using it would completely ignore `index.js` as you've said.